src/util/scrollTo.tsx

```diff
@@ -1,15 +1,20 @@
+import { isMobile } from './isDevice';
```

Comment:
We can use `isMobileScreen` from [`./isScreenWithinWidth.ts`](https://github.com/nodejs/nodejs.dev/blob/master/src/util/isScreenWithinWidth.ts#L22) for the same.